runtime/mainnet/src/lib.rs

```diff
@@ -667,6 +691,10 @@ mod runtime {
 	#[runtime::pallet_index(33)]
 	pub type MessageQueue = pallet_message_queue::Pallet<Runtime>;
 
+	// Contracts (using pallet-revive)
+	#[runtime::pallet_index(40)]
+	pub type Contracts = pallet_revive::Pallet<Runtime>;
```

Comment:
Whilst this would be my preference, there is a risk that tooling does not support this (e.g. Polkadot JS which loads the existing contracts UI but expects wasm contracts). Determining support for this approach might be important before any launch.

Reply:
really good point. Sounds like it would just be more headache than it's worth.

Reply:
Hopefully new tooling will do a better job with this sort of things. I believe some version of the ledger app also suffered from something similar to this with the balances pallet.

Reply:
Probably worthwhile to open a forum post

Reply:
https://github.com/r0gue-io/pop-node/pull/427/commits/5971736c7d3eec99ecf9421872ad4394e0b433e9